Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -402,6 +427,30 @@ final public class LottieAnimationView: LottieAnimationViewBase {
     }
   }
 
+  /// Sets the lottie file backing the animation view. Setting this will clear the
+  /// view's contents, completion blocks and current state. The new animation will
+  /// be loaded up and set to the beginning of its timeline.
+  /// The loopMode, animationSpeed and imageProvider will be set according
+  /// to lottie file settings
+  /// - Parameters:
+  ///   - dotLottie: Lottie file to play
+  ///   - animationId: Internal animation id to play. Optional
+  ///   Defaults to play first animation in file.
+  public func setDotLottie(
+    _ dotLottie: DotLottie?,
+    animationId: String? = nil)
```

Comment:
I wonder if a name like this would be more idiomatic:

```suggestion
  public func loadAnimation(
    _ animationId: String? = nil,
    from _ dotLottieFile: DotLottie)
```

I'd like to avoid "set" if possible. Open to other ideas as well.